### ISSUE TITLE
Patient und alle Rezepte löschen

### ISSUE THREAD

**Issue by zdavatz** [zdavatz/amiko_csharp#64]
Wenn man einen Patient und alle Rezept löschen möchte, muss man noch gefragt werden:

<img width="682" alt="bildschirmfoto 2018-06-04 um 13 41 58" src="https://user-images.githubusercontent.com/4953/40915656-2ce37b3e-67fd-11e8-9dd0-c8b2a85610a1.png">

Wenn man "Ja" klickt, dann wird der Adressbuch-Eintrag zusammen mit allen dazugehörigen Rezepten gelöscht.

**Comment by grauwoelfchen** [comment]
@zdavatz dann, machen wir das Confirmation Dialog zusammen, wenn wir den Löschen für die Rezepten machen. (mit Issue #69)

**Comment by zdavatz** [comment]
Ich denke es wäre besser, es jetzt zu machen weil wir jetzt am Patienten arbeiten.

**Comment by grauwoelfchen** [comment]
Dieses Dialog (eigentlich) sagt, dass den Löschen nicht nur für Kontakt ist, sondern es auch für Rezepte von ihm zusammen ist, wie du in Description erklärst. Ich denke, es für Rezepte ist.

Und ich denke, es vielleicht viel besser wäre, dass wir den Satz aktualisieren.

**Comment by zdavatz** [comment]
Ja, ich werde den Satz aktualisieren. Auf iOS ist er korrekt.

**Comment by zdavatz** [comment]
Der neue Satz wäre dann: "Alle Rezepte von/und PatientIn löschen?" Genau so.

**Comment by grauwoelfchen** [comment]
@zdavatz Ich verwende den neuen Satz.

**Comment by zdavatz** [comment]
Die FR Version lautet: "Supprimer le patient y compris les ordonnances resp."

**Comment by grauwoelfchen** [comment]
@zdavatz OK. Ich setze den Satz auf FR. Aber es is momentan so in iOS Version:
https://github.com/zdavatz/AmiKo-iOS/blob/master/AmiKoDesitin/fr.lproj/Localizable.strings#L20

**Comment by zdavatz** [comment]
Ja, der geht auch. Der ist einfach kürzer wegen dem Screeplatz auf iOS. ;). Du kannst auch den verwenden. Kein Problem.

**Comment by zdavatz** [comment]
Das klappt sehr gut mit dem Löschen.
